src/org/opensolaris/opengrok/analysis/CommonLaxFPath.lexh

```diff
@@ -0,0 +1,28 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2017, Chris Fraire <cfraire@me.com>.
+ */
+
+/*
+ * Relaxed from {FPath} in that only the last path segment is required
+ * to end in {ASCII_ALPHA} or {DIGIT}.
+ */
+LaxFPath = "/"? [a-zA-Z]{FNameChar}* ("/" [a-zA-Z]{FNameChar}*)+[a-zA-Z0-9]
```

Comment:
LaxFPath is kinda confusing name, would it make sense to call it e.g.
LastSegmentFPath ?

Reply:
I'm not sure about "LastSegmentFPath". It's really the middle that has been relaxed from {FPath}; would "RelaxedMiddleFPath" do?

Reply:
yeah, better than Lax (which stands for Relaxed? )

Reply:
Well "lax" is an adjective (Webster's:  "Not strict or stringent; not exact; loose; weak; vague; equivocal.") and "relax" is a verb ("to make lax") — but "relaxed" is an adj. too. I'll push that change.